airbyte-webapp/src/locales/en.json

```diff
@@ -335,6 +336,7 @@
   "connection.transfer": "Transfer",
   "connection.linkCopied": "Link copied!",
   "connection.copyLogLink": "Copy link to log",
+  "connection.connectionDeletedView": "This connection has been deleted.  This page is for reference only.  No changes or syncs can be completed.",
```

Comment:
Was this wording run by a PM?

Reply:
Nope!  Should be.  Part of why it is not ready for merge.  What is the process I should take for that?

Reply:
@andyjih open to input on wording!  This will be used in a banner as seen in the screenshots in the PR description.

Reply:
👍